### ISSUE TITLE
Swiftlint errors not showing in CI

### ISSUE THREAD

**Issue by Liquidsoul** [SwiftGen/SwiftGen#296]
I have just run locally the build and it fails on some enum names (PR incoming).
I would not have filed an issue if CircleCI was reporting the errors but it appears it does not.
We need to look into this and find why.

Here follows the errors when running `rake lint:code` at the root of the project for the current master (c1a6e48266ddef02fd1bbd36a9fab65500145fdb) which are different from what the [CI report contain](https://circleci.com/gh/SwiftGen/SwiftGen/112).

```
== Linting the code ==
swiftlint lint --no-cache --strict --path Sources
Loading configuration from '.swiftlint.yml'
Linting Swift files at path Sources
Linting 'ArgumentsUtils.swift' (1/8)
Linting 'colors.swift' (2/8)
Linting 'images.swift' (3/8)
Linting 'fonts.swift' (4/8)
Linting 'main.swift' (5/8)
Linting 'storyboards.swift' (6/8)
Linting 'templates.swift' (7/8)
Linting 'strings.swift' (8/8)
SwiftGen/SwiftGen/Sources/ArgumentsUtils.swift:44:8: error: Identifier Name Violation: Enum element name should start with a lowercase character: 'Console' (identifier_name)
SwiftGen/SwiftGen/Sources/ArgumentsUtils.swift:45:8: error: Identifier Name Violation: Enum element name should start with a lowercase character: 'File' (identifier_name)
SwiftGen/SwiftGen/Sources/ArgumentsUtils.swift:81:8: error: Identifier Name Violation: Enum element name should start with a lowercase character: 'NamedTemplateNotFound' (identifier_name)
SwiftGen/SwiftGen/Sources/ArgumentsUtils.swift:82:8: error: Identifier Name Violation: Enum element name should start with a lowercase character: 'TemplatePathNotFound' (identifier_name)
Done linting! Found 4 violations, 4 serious in 8 files.
rake aborted!
Command failed with status (2): [swiftlint lint --no-cache --strict --path ...]
SwiftGen/SwiftGen/rakelib/utils.rake:64:in `plain'
SwiftGen/SwiftGen/rakelib/utils.rake:18:in `run'
SwiftGen/SwiftGen/rakelib/lint.rake:22:in `block (2 levels) in <top (required)>'
Tasks: TOP => lint:code
(See full trace by running task with --trace)
```

**Comment by AliSoftware** [comment]
Probably because CircleCI doesn't have the same version of SwiftLint that we do on each of our machines?

**Comment by djbe** [comment]
Yeah CI is running an older version, and recently stencilswift started throwing errors for identifiers. @Liquidsoul already made a fix in swiftgenkit I think? Needs to be done for the other repos as well.

**Comment by diogot** [comment]
I'm using SwiftLint with CocoaPods in some projects, works very well.
I can do a PR if you think is fine.

**Comment by djbe** [comment]
That's actually not recommended, because we check the `Pods` directory into git.

I've opened a dedicated issue for this here: https://github.com/SwiftGen/Eve/issues/8

**Comment by Liquidsoul** [comment]
@djbe Yup I fixed the swiftlint issues in swiftgenkit here https://github.com/SwiftGen/SwiftGenKit/pull/32

**Comment by djbe** [comment]
Fixed by #297.

Further discussion on how to prevent swiftlint version issues can go in https://github.com/SwiftGen/Eve/issues/8.
